### PR TITLE
rgba() is not necessarily legacy.

### PR DESCRIPTION
**Pull request by estelle**
rgba() is an alias for rgb().

the example isn't using the legacy syntax of using commas, so i updated the text, title, and live sample function call to remove the "legacy" lingo.

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks, Estelle - taking a look now 👀

File: files/en-us/web/css/color_value/rgb/index.md
Comment (bsmth) [inline files/en-us/web/css/color_value/rgb/index.md:1]: There's a note on L12:

> **Note:** The legacy `rgba()` syntax is an alias for `rgb()`, accepting the same parameters and behaving in the same way.

We can remove "legacy" there, too:

```diff
-> **Note:** The legacy `rgba()` syntax is an alias for `rgb()`, accepting the same parameters and behaving in the same way.
+> **Note:** The `rgba()` syntax is an alias for `rgb()` - it accepts the same parameters and behaves the same way.
```

**Comment by bsmth** [comment]
files/en-us/web/accessibility/understanding_colors_and_luminance/index.md has a couple of places to fix:

L69:

```css
/* legacy rgba notation */
color: rgba(100%, 0%, 100%, 100%);
color: rgba(255, 0, 255, 1);
```

L96 has a broken xref and uses "legacy" wording (maybe you have better phrasing for this):

```diff
-The examples also show the legacy [`rgba()`](/en-US/docs/Web/CSS/color_value/rgb#legacy_syntax_rgba) syntax. The older syntax for color functions was comma-separated, with separate functions for when the alpha channel was included. New color functions only have one syntax with space-separated (rather than comma-separated) values, with the alpha channel, if present, being preceded by a slash.
+The examples also show the [`rgba()`](/en-US/docs/Web/CSS/color_value/rgb#rgba_syntax) syntax which is an alias for `rgb()`- it accepts the same parameters and behaves the same way. The legacy syntax for color functions is comma-separated and relies on using separate functions when you need to use the alpha channel. New color functions use space-separated values, and a slash precedes an optional alpha channel.
```

**Review by bsmth — CHANGES_REQUESTED** [review]
Thanks Estelle! A couple of comments for you to consider 🙌🏻

File: files/en-us/web/css/color_value/rgb/index.md
Comment (estelle) [reply files/en-us/web/css/color_value/rgb/index.md:1]: that was fixed in a different PR that was merged today ;)

**Comment by estelle** [comment]
Thanks @bsmth 
I changed much more than that line to be in line with #32004 that was merged earlier today.

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md [hunk 1]
```diff
@@ -109,17 +123,15 @@ In addition to developer tools, many tools can convert RGB to HSL for you and pr
...
 
 Where accessibility is concerned, however, standards and guidelines are currently written predominantly using the sRGB color space, especially as it applies to color contrast ratios.
 
 > **Note:** Almost all systems used today to view Web content assume sRGB encoding. Unless it is known that another color space will be used to process and display the content, authors should evaluate using sRGB colorspace. If using other color spaces, apply the principles of [minimum contrast ratios](https://webaim.org/articles/contrast/#sc143).
 
-[CSS Color Module Level 5](https://drafts.csswg.org/css-color-5/) introduced [`lch()`](/en-US/docs/Web/CSS/color_value/lch) functional color notation and the [`lab()`](/en-US/docs/Web/CSS/color_value/lab) color coordinate system which can specify any visible color. These will change the domination of the sRGB color space in the future, but we are not there yet.
```
Comment (estelle) [inline files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:118]: I removed this as we are there, we don't need to point out level 5, and the mentioned functions were added to the content above.

**Comment by bsmth** [comment]
> Thanks @bsmth I changed much more than that line to be in line with #32004 that was merged earlier today.

Sure thing, taking a look now 👀

File: files/en-us/web/css/color_value/rgb/index.md
Comment (bsmth) [reply files/en-us/web/css/color_value/rgb/index.md:1]: Super, ty

File: files/en-us/web/accessibility/understanding_colors_and_luminance/index.md (hunk 1 above)
Comment (bsmth) [reply files/en-us/web/accessibility/understanding_colors_and_luminance/index.md:118]: nice tnx

**Comment by bsmth** [comment]
There's two other places saying "legacy <rgba()>", we might want to get those, too:

In CSS Values and Units (files/en-us/web/css/css_values_and_units/index.md):

>Some legacy functional notations such as `rgba()` use commas,

In `files/en-us/web/css/color/index.md`:

```css
/* <rgb()> values and legacy <rgba()> values*/
color: rgb(34, 12, 64, 0.6);
color: rgba(34, 12, 64, 0.6);
color: rgb(34 12 64 / 0.6);
color: rgba(34 12 64 / 0.6);
color: rgb(34.6 12 64 / 60%);
color: rgba(34.6 12 64 / 60%);
```

I don't know if something like this helps, it would be nice to use the same format for the rest of the functions:

```css
/* <rgb()> and <rgba()> values*/
color: rgb(34 12 64);
color: rgba(34 12 64 1);
/* with an alpha channel */
color: rgb(34.6 12 64 / 60%);
color: rgba(34.6 12 64 / 60%);
/* legacy comma-separated syntax */
color: rgb(34, 12, 64, 0.6);
color: rgba(34, 12, 64, 0.6);
```

What do you think?

**Comment by bsmth** [comment]
Thanks, Estelle, two files for us to check with 'legacy' wording, but passing it back over to you to have a look!

**Pull request closed without merging**